src/evt/script_list.c

```diff
@@ -311,7 +311,7 @@ Evt* start_child_script(Evt* parentScript, EvtSource* source, s32 initialState)
     parentScript->childScript = child;
     parentScript->state |= 0x10;
     child->state = initialState | 1;
-    child->ptrCurrentLine = child->ptrFirstLine = child->ptrNextLine = source;
+    child->ptrCurrentLine = child->ptrFirstLine = child->ptrNextLine = (Bytecode*)source;
```

Comment:
Hmm all these kinda pointless casts make me think that Bytecode and EvtSource should be the same type...
Or we retype all Evt's `line` fields to `EvtSource`; then we can think of Bytecode as the atom and EvtSource as a pointer to a `{ opcode, argc, ...argv }` (where opcode, argc, and argv[] are all Bytecodes).

It's past 1am so that probably made no sense but I think I may try redefining/renaming these types in a later PR 😛  
We could also drop the distinction entirely but I think its nice-to-have

Reply:
I'd be more down to drop the distinction personally, but yeah I get what you're saying